src/proxy/NativeMessagingHost.cpp

```diff
@@ -22,12 +22,13 @@
 #include <Winsock2.h>
 #endif
 
-NativeMessagingHost::NativeMessagingHost() : NativeMessagingBase(true)
+NativeMessagingHost::NativeMessagingHost()
+    : NativeMessagingBase(true)
 {
     m_localSocket = new QLocalSocket();
     m_localSocket->connectToServer(getLocalServerPath());
     m_localSocket->setReadBufferSize(NATIVE_MSG_MAX_LENGTH);
-  
```

Comment:
Please remove all these files from the diff which have no changes except minor formatting.